code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1226,6 +1226,7 @@
 		RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_JANITOR
 	)
 
+
```

Comment:
nitpick but unneeded space

Reply:
Bah, I hate those too. Lemme get it.

Reply:
I go to delete the line, and VSCode just keeps adding it back automatically, what the hell.